SKILL.tsv

```diff
@@ -1775,7 +1775,7 @@ SKILL_20150317_001774	HP Aurora
 SKILL_20150317_001775	HP recovery is improved.
 SKILL_20150317_001776	Size has shrunk.
 SKILL_20150317_001777	Parrot
-SKILL_20150317_001778	Ignore evasion of enemy but chance of critical and critical evasion is decreased.
+SKILL_20150317_001778	Your attacks are guaranteed to hit, increasing your Critical Rate and Critical Resistance.
```

Comment:
Still, the description is wrong. "Aggressor" might replaced with a new skill name.